lib/manager/sbt/extract.ts

```diff
@@ -176,6 +176,10 @@ function parseDepExpr(
     currentValue,
   };
 
+  if (variables[rawVersion]) {
+    result.groupName = `${rawVersion} for ${groupId}`;
```

Comment:
It's hard to separate by package file, but maybe it's good enough to group just with single `groupId`

Reply:
IMHO it's good enough to group by `groupId` ;)

Reply:
@viceice @rarkins What grouping criteria would you prefer?

Reply:
`groupId` sounds good to me

Reply:
Okay, so I'm leaving it as-is, i.e. `variable + groupId`

Reply:
🤔 ok